core/vm/opcodes.go

```diff
@@ -198,6 +198,11 @@ const (
 	LOG4
 )
 
+// account Abstraction ops.
+const (
+	PAYGAS OpCode = 0xaa + iota
```

Comment:
should it be 0x49 as in the eip description https://eips.ethereum.org/EIPS/eip-2938 ?

Reply:
this was taken from the original upstream implementation. I can change as per EIP specification. what is the rule here, should we follow upstream to be bytecode compatible or follow the specs?

Reply:
we should follow the upstream implementation for now.